client/js/table-view.js

```diff
@@ -73,6 +75,22 @@ class TableView {
     this.sheetBodyEl.appendChild(fragment);
   }
 
+  renderTableFooter() {
+    const fragment = document.createDocumentFragment();
```

Comment:
Even better if you don't need to make a new document fragment to render the table footer. Check out the renderTableHeader function and how it's not using the fragment technique.